scripts/build-binary.ps1

```
@@ -0,0 +1,61 @@
+$ScriptDir = Split-Path -Parent $MyInvocation.MyCommand.Path
+$RepoDir = Split-Path -Parent $ScriptDir
+$VenvDir = Join-Path $ScriptDir ".venv"
+$DistDir = Join-Path $ScriptDir "dist"
+
+Write-Host "[build] Building aicp standalone binary"
+Write-Host "[build] Repo: $RepoDir"
+
+# Use python from PATH or PYTHON env var
+$Python = if ($env:PYTHON) { $env:PYTHON } else { "python" }
+$PyVersion = & $Python -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')"
+Write-Host "[build] Using Python $PyVersion"
```

PSScriptAnalyzer (1.23.0)

[warning] File 'build-binary.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)